scripts/docker

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
-IMAGE=NodeOS
-USERSFS=out/latest/usersfs
+IMAGE=nodeos
+USERSFS=/out/latest/usersfs
```

Comment:
Ok, it seems that if it doesn't start by a slash it think we are going to use a Docker volume... What a sh*t.